packages/confirmation-modal/src/ConfirmationModal.tsx

```diff
@@ -46,11 +46,11 @@ const buttonStyle = css`
   margin: 0 2px;
 
   :first-of-type {
```

Comment:
Nit: I know this selector wasn't written in this PR, but we should include the `&` for this sort of selector to indicate if it's being applied to the current className or if it's a descendent selector.

```
& :first-of-type {
  margin: 0 0 0 4px;
}
```

Also, do we need to be defining the margin on all 4 sides here, or would a simple margin-left be sufficient?

Reply:
nope, i think simple margin-left is sufficient here

Reply:
but added the & selector